CodeFormatter/codeformatter.h

```diff
@@ -30,94 +30,74 @@
 #include "formatoptions.h"
 #include "plugin.h"
 
+enum CodeFormatterEngine {
+    cFormatEngineNone,
+    cFormatEngineAStyle,
+    cFormatEngineClangFormat,
+    cFormatEngineBuildInPhp,
+    cFormatEnginePhpCsFixer,
+    cFormatEnginePhpcbf,
+    cFormatEngineWxXmlDocument,
+};
+
```

Comment:
In all the new code of CodeLite, enumerators starts with `k`, so it should be `kFormatEngineNone`, `kFormatEngineAStyle` etc